test/unit/modules/compiler/parser.spec.js

```diff
@@ -123,7 +123,7 @@ describe('parser', () => {
   it('not warn 3 root elements with v-if, v-else-if and v-else', () => {
     parse('<div v-if="1"></div><div v-else-if="2"></div><div v-else></div>', baseOptions)
     expect('Component template should contain exactly one root element')
-        .not.toHaveBeenWarned()
```

Comment:
these lines were throwing `Expected indentation of 6 spaces but found 8 indent`.